### ISSUE TITLE
Wrong depends_on in docker-compose.yml when not chosing postgres or redis

### ISSUE THREAD

**Issue by rmehner** [hintmedia/railsdock#8]
:wave:

I stumbled over this little bug (that's easy to fix manually in the result file, so no biggie):

If I select the "mysql" and "memcached", the docker-compose file still depends on Postgres and Redis.

Resulting docker-compose.yml includes this:

```yml
depends_on:
    - postgres
    - redis
```

but should include this:

```yml
depends_on:
    - mysql
    - memcached
```

instead.

Thank you for reading and have a great day!

**Comment by nvick** [comment]
Thanks for opening an issue, @rmehner! We will dig into it soon!

**Comment by kyleboe** [comment]
Got this resolved and released version `0.2.0`! Thanks for reporting the bug 👍
